skeletons/function.xml

```diff
@@ -108,31 +110,31 @@ Use the PEAR Coding standards
 Use a screen like we did below
 ]]>
    </screen>
-  </example><!-- }}} -->
- </refsect1><!-- }}} -->
+  </example>
+ </refsect1>
```

Comment:
Thanks for removing the folding markers. :)

Reply:
Should we add a note to the style guide that these are discouraged?

Reply:
Probably yes